### PR TITLE
#2229 - Remove Exceptional Expenses from PT assessment

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Removed `calculatedDataExceptionalExpenses` from PT assessment;
- Checked that `calculatedDataExceptionalExpenses` was only used for PT assessments;

### CODE REVIEW

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:modeler="http://camunda.org/schema/modeler/1.0" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" id="Definitions_0ihicdz" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.18.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.1.0" camunda:diagramRelationId="7a75e87a-6774-4e60-9a94-5dbbe3be143a">
```
Comment (dheepak-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn:2]: 👍

**Review by dheepak-aot — APPROVED** [review]
Nice work @andrepestana-aot 👍

**Review by lewischen-aot — APPROVED** [review]
LGTM nice work 👍

**Review by ann-aot — APPROVED** [review]
LGTM 👍 nice work @andrepestana-aot

**Pull request merged**